getting-started/kubernetes/trying-ebpf.md

```diff
@@ -0,0 +1,228 @@
+- [Install Calico on nodes](#install-calico-on-nodes)
+- [Toggle between eBPF and the standard linux networking pipeline](#toggle-between-ebpf-and-the-standard-linux-networking-pipeline)
+
+#### Set up a suitable cluster
+
+We recommend using `kubeadm` to bootstrap a suitable cluster on AWS.  
+
+1. In AWS, create a controller node and at least 2 worker nodes in the same VPC subnet.  Use Ubuntu 19.10 as the image.
+
+1. **Important** disable the source/destination check on each node's interface.  This is required in order to use non-overlay networking.
+
+   1. Open the Amazon EC2 console and navigate to "Instances".
+
+   1. Select the instance and choose **Actions** > **Networking** > **Change Source/Dest. Check**.
+
+   1. Verify that source/destination check is disabled. Otherwise, choose **Yes, Disable**.
+
+1. On each node, {% include open-new-window.html text='install kubeadm' url='https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/' %}; we recommend using docker as the containeriser in order to match our test environment as closely as possible.
+
+1. Create the controller node of a new cluster. On the controller VM, execute:
+
+   ```
+   sudo kubeadm init --pod-network-cidr 192.168.0.0/16
+   ```
+
+1. To set up kubectl for the ubuntu user, run:
+
+   ```
+   mkdir -p $HOME/.kube
+   sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+   sudo chown $(id -u):$(id -g) $HOME/.kube/config
+   ```
+   The final line of the kubeadm init output contains the command for joining your workers to the controller.  Run this on each worker, prepending `sudo` to run it as root.  It will look something like this:
+
+   ```
+   sudo kubeadm join 10.240.0.11:6443 --token <token> --discovery-token-ca-cert-hash sha256:<hash>
+   ```
+
+1. On the controller, verify that all nodes have joined
+
+   ```
+   kubectl get nodes
+   ```
+   which should output something similar to:
+
+   ```
+   NAME                                           STATUS      ROLES    AGE   VERSION
+   ip-172-16-101-119.us-west-2.compute.internal   NotReady    <none>   86m   v1.17.3
+   ip-172-16-101-27.us-west-2.compute.internal    NotReady    <none>   86m   v1.17.3
+   ip-172-16-101-49.us-west-2.compute.internal    NotReady    master   88m   v1.17.3
+   ip-172-16-102-174.us-west-2.compute.internal   NotReady    <none>   86m   v1.17.3
+   ```
+
+   The nodes will report NotReady until we complete the installation of {{site.prodname}} below.
+
+#### Install {{site.prodname}} on nodes
+
+For the tech preview, only the Kubernetes API Datastore is supported.  Since {{site.prodname}} replaces `kube-proxy` in eBPF mode, it requires the IP and port of your API server to be set in its config map. 
+
+1. Download the {{site.prodname}} networking manifest for the Kubernetes API datastore.
+
+   ```bash
+   curl {{ "/manifests/calico-bpf.yaml" | absolute_url }} -O
+   ```
+{% include content/pod-cidr-sed.md yaml="calico" %}
```

Comment:
I don't think the pod-cidr-sed instructions make sense anymore since we've added auto-detection for kubeadm, and commented that section by deafult.

Since this guide is using kubeadm, I think we can remove entirely, but we'll need to (separately) update those instructions to make sense for Calico v3.13.

@rafaelvanoni can you own that? It's a follow on from the auto-detection work you did.